blocks_extras/extensions/synchronization.py

```diff
@@ -137,15 +137,6 @@ def seed(self):
     def init_shared_params(self, parameters):
         super(SynchronizeWorker, self).init_shared_params(
             parameters, self.sync_rule)
-        if self.is_main_worker:
```

Comment:
We should go through all the documentation and make sure that it does not mention this initialization trick